code/game/machinery/bots/mulebot.dm

```diff
@@ -782,6 +782,7 @@ var/global/mulebot_count = 0
 	B.add_blood_list(H)
 	add_blood_list(H)
 	bloodiness += 4
+	add_logs(H, src, "ran over")
```

Comment:
Yeah I'm retarded. Is it possible to get the user who's riding the mulebot then?